README.md

```diff
@@ -15,142 +24,26 @@ website][tf-website] and the [GitHub project page][tf-github].
-anything within the 1.x namespace, including or after 1.5.0. [Read
-more][provider-vc] on provider version control.
-
-[provider-vc]: https://www.terraform.io/docs/configuration/providers.html#provider-versions
-
-# Building The Provider
-
-**NOTE:** Unless you are [developing](#developing-the-provider) or require a
-pre-release bugfix or feature, you will want to use the officially released
-version of the provider (see [the section above](#using-the-provider)).
-
-## Cloning the Project
-
-First, you will want to clone the repository to
-`$GOPATH/src/github.com/terraform-providers/terraform-provider-vsphere`:
-
-```sh
-mkdir -p $GOPATH/src/github.com/terraform-providers
-cd $GOPATH/src/github.com/terraform-providers
-git clone git@github.com:terraform-providers/terraform-provider-vsphere
-```
-
-## Running the Build
-
-After the clone has been completed, you can enter the provider directory and
-build the provider.
-
-```sh
-cd $GOPATH/src/github.com/terraform-providers/terraform-provider-vsphere
-make build
-```
-
-## Installing the Local Plugin
-
-After the build is complete, copy the `terraform-provider-vsphere` binary into
-the same path as your `terraform` binary, and re-run `terraform init`.
-
-After this, your project-local `.terraform/plugins/ARCH/lock.json` (where `ARCH`
-matches the architecture of your machine) file should contain a SHA256 sum that
-matches the local plugin. Run `shasum -a 256` on the binary to verify the values
-match.
-
-# Developing the Provider
-
-**NOTE:** Before you start work on a feature, please make sure to check the
-[issue tracker][gh-issues] and existing [pull requests][gh-prs] to ensure that
-work is not being duplicated. For further clarification, you can also ask in a
-new issue.
-
-[gh-issues]: https://github.com/terraform-providers/terraform-provider-vsphere/issues
-[gh-prs]: https://github.com/terraform-providers/terraform-provider-vsphere/pulls
-
-If you wish to work on the provider, you'll first need [Go][go-website]
-installed on your machine (version 1.9+ is **required**). You'll also need to
-correctly setup a [GOPATH][gopath], as well as adding `$GOPATH/bin` to your
-`$PATH`.
-
-[go-website]: https://golang.org/
-[gopath]: http://golang.org/doc/code.html#GOPATH
-
-See [Building the Provider](#building-the-provider) for details on building the provider.
-
-# Testing the Provider
-
-**NOTE:** Testing the vSphere provider is currently a complex operation as it
-requires having a vCenter endpoint to test against, which should be hosting a
-standard configuration for a vSphere cluster. Some of the tests will work
-against ESXi, but YMMV.
-
-## Configuring Environment Variables
+## Requirements
+-	[Terraform](https://www.terraform.io/downloads.html) 0.12.x
+    - Note that version 0.11.x currently works, but is [deprecated](https://www.hashicorp.com/blog/deprecating-terraform-0-11-support-in-terraform-providers/)
+- vSphere 6.7    
```

Comment:
I think we should maintain support aligned with the [VMware product lifecycle matrix](https://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/support/product-lifecycle-matrix.pdf), which would mean supporting 6.5 until Nov. 15, 2021.

Reply:
Sounds good to me. @bill-rich @koikonom any concerns?

Reply:
It would be difficult for us to actually test against multiple versions at this point, but I do think it would be good to align with VMware's lifecycle. I would say we will support it, but it may be more difficult for us to troubleshoot, and the chances are higher that a bug could make it through testing that would affect earlier versions.